docs/README.md

```diff
@@ -638,6 +642,44 @@ will apply migrations from `41_add_enum_column` onwards to the target database.
 
 If the `--complete` flag is passed to `pgroll migrate` the final migration to be applied will be completed. Otherwise the final migration will be left active (started but not completed).
 
+### Latest
+
+`pgroll latest` prints the latest version in a directory of migration files, optionally prefixing the version with the current schema.
+
+Assuming that the [example migrations](https://github.com/xataio/pgroll/tree/main/examples) are on disk in a directory called `examples`, running:
+
+```
+$ pgroll latest examples/
+```
+
+will print the latest migration in the directory:
+
+```
+45_add_table_check_constraint
+```
+
+If the `--with-schema` flag is passed to `pgroll latest` the latest version will be prefixed with the current schema:
+
+```
+public_45_add_table_check_constraint
+```
+
+The current schema can be set with the global `--schema` flag:
+
+```
+$ pgroll latest examples/ --with-schema --schema foo
+```
+
+This prints:
+
+```
+foo_45_add_table_check_constraint
+```
+
+The `pgroll latest` command is useful in scripts that need to retrieve the latest version from a set of migrations files, for example to set the Postgres `search_path` in a CD workflow.
```

Comment:
I wonder if we need to be more explicit. Is this the last migration that was completed? Or is this the last in progress migration?

Reply:
It's actually neither. 

The command prints the latest version it finds in the argument directory of migrations. That migration may not have even been applied to the database yet.

The use case is to run this command in a CD workflow where the service to be deployed needs to set the Postgres `search_path` as part of the DB connection string. The idea is that somewhere in the workflow this is run:

```
DB_SCHEMA=$(pgroll latest --with-schema migrations/)
// $DB_SCHEMA now contains something like "public_03_add_constraint"
```

then the connection string for the application can be built using `$DB_SCHEMA`.

Maybe `pgroll latest` is not the best name for this command, but I'm not sure of a better name.

Reply:
Yes, it is not really clear. Either way I do not yet have a better name for this. Instead we could document it a bit more verbosely. Could you please add a note about this to the documentation? I assumed based on reading the code that it has nothing to do with the applied migrations. But I would prefer that users avoid this step. :)

Reply:
I've updated the command so that it can now retrieve the latest schema version from either the target database or a local migrations directory.

I think this addresses the confusion about what the command is operating on.